lms/services/file.py

```diff
@@ -21,6 +21,31 @@ def get(self, lms_id, type_):
             .one_or_none()
         )
 
+    def get_with_guid(self, lms_id, type_):
+        """
+        Return with the given lms_id and type_ that belongs to the same GUID as the current ApplicationInstance.
+
+        Once we have "Organization" or a similar entity that groups together multiple
+        ApplicationInstance we should use that concept instead of GUID but for now
+        this allows us to find files across multiple installs in for example
+        migrations from course/account/global level install or in some LTI1.3 upgrade
+        scenarios.
```

Comment:
```suggestion
        Get matching files linked to this application instance by GUID.

        Once we have "Organization" or a similar entity that groups together
        multiple ApplicationInstance we should use that concept instead of GUID
        but for now this allows us to find files across multiple installs in,
        for example, migrations from course/account/global level install or in
        some LTI1.3 upgrade scenarios.
```

Some of the organisation info here seems more of a comment for future work than a description of what this method does?